includes/license.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+if ( ! defined( 'SS_LICENSE_SERVER' ) ) {
+    define( 'SS_LICENSE_SERVER', 'https://license.strangerstudios.com/' );
+}
+
+/**
+ * Add a monthly check to see if license key is still active.
+ */
+function sws_license_activation() {
+    sws_maybe_schedule_event( current_time( 'timestamp' ), 'monthly', 'sws_license_check_key' );
+}
+register_activation_hook( __FILE__, 'sws_license_activation' );
+
+/**
+ * Remove monthly check when plugin is deactivated.
+ */
+function sws_license_deactivation() {
+	wp_clear_scheduled_hook( 'sws_license_check_key' );
+}
+register_deactivation_hook( __FILE__, 'pmpro_deactivation' );
```

Comment:
Need to hook `'sws_license_deactivation'`

Reply:
Doh! Thanks!

Reply:
This has been adjusted.

Reply:
This has been updated.